src/components/AccountForm.vue

```diff
@@ -703,6 +703,19 @@ export default {
 	font-weight: bold;
 }
 
+:deep(.checkbox-radio-switch) {
+	border: 0 !important;
```

Comment:
this border is indended: https://nextcloud-vue-components.netlify.app/#/Components/NcCheckboxRadioSwitch

we have to fix that it's intermitted

Reply:
Yep